resources/serverless/templates/cluster-role-binding.yaml

```diff
@@ -10,5 +10,5 @@ roleRef:
   name: {{ template "fullname" . }}
 subjects:
 - kind: ServiceAccount
-  name: {{ template "fullname" . }}
+  name: {{ template "fullname" . }}-controller-manager
```

Comment:
If you changed serviceAccount name, please also change in values.yaml - this name is also used by Job for building Function image.